scalding-core/src/main/scala/com/twitter/scalding/Operations.scala

```diff
@@ -467,10 +467,10 @@ package com.twitter.scalding {
     }
 
     def complete(flowProcess: FlowProcess[_], call: AggregatorCall[X]): Unit = {
-      emit(flowProcess, call)
+      emit(call)
     }
 
-    def emit(flowProcess: FlowProcess[_], call: AggregatorCall[X]): Unit = {
+    def emit(call: AggregatorCall[X]): Unit = {
       call.getOutputCollector.add(set(call.getContext))
```

Comment:
well, you have changed the signature, so any old callers are broken. Keeping it with a new signature does not reduce breakage.

Let's not assume we have a hidden reason if there is no comment and no broken test with improving the code.